### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by ISRLv2021**
- [DEMO LINK](https://ISRLv2021.github.io/layout_search-bar-airbnb/)
- [TEST REPORT LINK](https://ISRLv2021.github.io/layout_search-bar-airbnb/report/html_report/)

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+.big-input {
+  width: 100%;
+  height: 70px;
+  font-size: 16px;
+  line-height: 22px;
+  padding-left: 62px;
+}
+
+.common-input {
+  font-family: Avenir-book, sans-serif;
```
Comment (Denchepornyuk) [inline src/style.css:24]: add font-family to html and use `inherit` here
https://developer.mozilla.org/en-US/docs/Web/CSS/inherit

**Review by Denchepornyuk — CHANGES_REQUESTED** [review]
good job

**Review by Denchepornyuk — CHANGES_REQUESTED** [review]
Did you push new code?

File: src/style.css (hunk 1 above)
Comment (ISRLv2021) [reply src/style.css:24]: Understandable

Comment (ISRLv2021) [reply src/style.css]: @font-face {
  font-family: "Avenir-book";
  src: url(./fonts/Avenir-Book.ttf);
}

@font-face {
  font-family: "Avenir-Heavy";
  src: url(./fonts/Avenir-Heavy.ttf);
}

html {
  font-family: Avenir, sans-serif;
  font-weight: 300px;
}

and after 
 .common-input {
  font-family: inherit;
}

Like this?

Comment (Denchepornyuk) [reply src/style.css]: yep, like this

**Review by Denchepornyuk — APPROVED** [review]

**Pull request closed without merging**